### ISSUE TITLE
Migrate places that read dendron-code.workspace via fs.readfile to filesv2.readJSONWithComments

### ISSUE THREAD

**Issue by bs** [dendronhq/dendron#304]
```
rejected promise not handled within 1 second: SyntaxError: /Users/britt/Documents/wikish.private/dendron.code-workspace: Unexpected token / in JSON at position 347
extensionHostProcess.js:1063
stack trace: SyntaxError: /Users/britt/Documents/wikish.private/dendron.code-workspace: Unexpected token / in JSON at position 347
	at JSON.parse (<anonymous>)
	at Object._readFile (/Users/britt/Documents/src/dendron/packages/plugin-core/node_modules/jsonfile/index.js:25:16)
```

**Comment by bs** [comment]
vscode follows json spec + json w/ comments (jsonc), which I don't even see a formal spec for.
`fs-extra` reads json via `jsonfile` which is actual json, so workspace configs can be valid in vscode and invalid in dendron.

Need to migrate these to the new [readJSONWithComments](https://github.com/dendronhq/dendron/blob/master/packages/common-server/src/filesv2.ts#L121:L121)

**Comment by kevinslin** [comment]
Will be updated in the next release!

**Comment by kevinslin** [comment]
fixed in 0.13.4!
